views.py

```diff
@@ -51,6 +56,26 @@ def get_object(self, *args, **kwargs):
 class EinWettbewerbKonkret(EinWettbewerb):
     model = models.WettbewerbKonkret
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        ich = self.get_object()
+        try:
+            context['fortsetzung'] = models.WettbewerbKonkret.objects.get(
+                wettbewerb=ich.wettbewerb.fortsetzung,
+                jahrgang=ich.jahrgang
+            )
+        except:
+            context['fortsetzung'] = False
+        context['vorherige'] = models.WettbewerbKonkret.objects.filter(
+            wettbewerb__fortsetzung=ich.wettbewerb,
+            jahrgang=ich.jahrgang
+        )
+        context['nachbarn'] = models.WettbewerbKonkret.objects.filter(
```

Comment:
muss das nicht auch in den try-Block rein?
wenn ich einmal dabei bin^^: am besten auch nach der letzten Zeile ein Komma, also zweimal nach ich.jahrgang

Reply:
filter liefert eine Liste aller Objekte auf die das zutrifft also brauche ich kein Try weil wenn es keine Gibt die Liste einfach Leer ist